src/scss/components/infinite_table.scss

```diff
@@ -10,4 +10,9 @@
 	    font-weight: bold !important;
 	    text-align: center;
 	}
+
+	&.no.show {

```

Comment:
use one-class name `no-show`, doing it otherwise is buggy. (Semantic made a very hard job to create combinations of classes while keeping the bugs out, but it's a lot of work and it's not really critical)

Reply:
The modules should use single class format like @2roy999 described. The words syntax is reserved for the semantic UI and the FLL UI packages, where they are checked to see they do not conincide any other classes.